.github/workflows/maven.yml

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Java CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        java: [8, 11, 14]
```

Comment:
We can keep just 8 and 11 for github actions.
We already have Jenkins with the full matrix.

Reply:
Thanks for the feedback! 
On the one hand I'm fine with removing 14 and only keeping the LTSes. On the other hand by keeping the latest non-LTS version we would reduce the risk of it failing on Jenkins eventually. The only downside I see is that we would have to upgrade this version number every half year. Am I missing other disadvantages?

Reply:
We don't make that much cost by adding the latest non-LTS here :-). I'd argue to keep it.

Reply:
After discussing this offline with @mthmulders and checking the ASF Jenkins build, we chose to run the build only on Java 8 on all platforms. We think the most value lies in running the integration tests against a wide variety of platforms and Java versions, so we introduced the matrix at the integration tests job. Interested in your thoughts on this @eolivelli .

Reply:
11 sounds good to have as well